src/platform-includes/getting-started-config/javascript.react.mdx

```diff
@@ -10,13 +11,21 @@ Sentry.init({
     new Sentry.BrowserTracing({
       // Set `tracePropagationTargets` to control for which URLs distributed tracing should be enabled
       tracePropagationTargets: ["localhost", /^https:\/\/yourserver\.io\/api/],
+      // See docs for support of different versions of variation of react router
+      // https://docs.sentry.io/platforms/javascript/guides/react/configuration/integrations/react-router/
```

Comment:
I think giving contextual info in a code snippet like this is pretty likely to be missed. I'll add a commit to this PR pulling this into the previous text.

Reply:
```suggestion
```

Reply:
I think it is good to add it above as you suggest, but I worry people will skip that as well, run to the copy paste, and in the end that's what goes in their code, and might be looked at by someone in their org for a PR review and so on

Searching all of GH in the past for different Sentry configs I see many users leave these comments and it is helpful when it links to docs with latest info, let's say when react-router v7 comes out and we have a new version they must update this line. Which is unfortunate design in SDK but not sure the constraints, and that is another rabbit hole :D

If you don't mind I would leave the comments for now, since it was also agreed with SEs to help with customer onboarding